src/ui/controls/SwitchControl.tsx

```diff
@@ -39,7 +42,7 @@ export class SwitchControl extends React.Component<IBooleanControlProps, void> {
 
   /** @override */
   componentDidUpdate() {
-    let materialSwitch = this.refs[CSS.RMX_SWITCH]["MaterialSwitch"];
+    let materialSwitch = this.switchControl["MaterialRadio"];
```

Comment:
Sorry I had previously fixed that misspelling.  Its changed to MaterialSwitch (not sure why its showing the old one here.